static/app/components/events/eventStatisticalDetector/breakpointChart.tsx

```diff
@@ -29,6 +29,52 @@ type EventBreakpointChartProps = {
   event: Event;
 };
 
+type BreakpointChartData = {
+  event: Event;
+  eventStatsReponse:
+    | {
+        data: EventsStatsData;
+        meta: MetaType;
+      }
+    | undefined;
+};
+function getBreakPointChartPropsFromData(chartData: BreakpointChartData) {
```

Comment:
Same in this function. Using react hooks like this can easily lead to bugs later.